apps/bandwagon/forms.py

```diff
@@ -136,6 +135,9 @@ class CollectionForm(ModelForm):
     icon = forms.FileField(label=_lazy(u'Icon'),
                            required=False)
 
+    # This is just a honeypot field for bots to get caught
+    your_name = forms.CharField(widget=forms.HiddenInput(), required=False)
```

Comment:
I don't think if this is just type="hidden" it will work as a honeypot field. It's likely to get ignored by a bot. Typically a honeypot field should be a normal field hidden with CSS. It should also be labelled in such a way to make sure people with assistive tech know to not fill it in.